app-creator/src/widgets/test/app-root_test.ts

```diff
@@ -0,0 +1,24 @@
+import {AppRoot} from '../app-root.js';
+import {fixture, html} from '@open-wc/testing';
+
+const assert = chai.assert;
+
+suite('app-root', () => {
+  test('is defined', () => {
+    const el = document.createElement('app-root');
+    assert.instanceOf(el, AppRoot);
+  });
+
+  test('renders with default values', async () => {
+    const el = await fixture(html`<app-root></app-root>`);
+    assert.shadowDom.equal(
+      el,
+      `
+      <div id="app">
+        <tool-bar></tool-bar>
+        <div id="container"></div>
+      </div>
+    `
+    );
```

Comment:
This test is brittle: if you decide to change the ordering of the elements, or add new ones, the test will break even though there's no error. Instead, you can just test that the `app-root` element has at least 1 child, which sufficiently tests that it initialized properly. Same below.